srcpkgs/olm/template

```diff
@@ -1,24 +1,17 @@
 # Template file for 'olm'
 pkgname=olm
-version=2.3.0
+version=3.0.0
 revision=1
-build_style=gnu-makefile
-make_check_target=test
+build_style=cmake
+configure_args="-DOLM_TESTS=OFF"
 hostmakedepends="python-devel python3-devel"
 makedepends="python-devel python3-devel"
-short_desc="An implementation of the Double Ratchet cryptographic ratchet"
+short_desc="Implementation of the Double Ratchet cryptographic ratchet"
 maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
 license="Apache-2.0"
 homepage="https://git.matrix.org/git/olm/about/"
 distfiles="https://git.matrix.org/git/olm/snapshot/${pkgname}-${version}.tar.gz"
-checksum=533714fb84860e04c185790d16ef9085f15e902c2105db941d5c7e92b0565ef8
-
-post_configure() {
-	# -O0 prevents _FORTIFY_SOURCE working
-	for _f in $(find ${wrksrc} -name Makefile); do
-		sed -i $_f -e "s; -O0;;"
-	done
-}
+checksum=4c54f93e4f426a9d5c8e66e63113d42fd2afca240cdcff30a469a342a6ec3bc6
 
 do_install() {
 	make DESTDIR=${DESTDIR} PREFIX=/usr install
```

Comment:
The cross-builds fail because of the custom `do_install()`. `$CC` seems to be lost/ignored, but when setting it explicitly it works:

    make CC=${CC} DESTDIR=${DESTDIR} PREFIX=/usr install

Reply:
ignored would be more precise. `make install` should not need to compile anything, Imo `do_install` should be the one from the build_style if possible and everything else should be compiled already before. But if there is no cleaner solution just go with explicitly with `CC=${CC}` as suggested above.